pkg/testutil/env/env.go

```diff
@@ -19,6 +19,8 @@ package env
 import (
 	"flag"
 	"fmt"
+	"os"
```

Comment:
Not related, but since I was mucking around in here a bit earlier, I figured I'd just leave this in